ffv1.md

```diff
@@ -233,15 +233,27 @@ RFC:```
 
 If the context is smaller than 0 then -context is used and the difference between the sample and its predicted value is encoded with a flipped sign.
 
-## Quantization
+## Quantization tables
 
-There are 5 quantization tables for the 5 sample differences, both the number of quantization steps and their distribution are stored in the bitstream. Each quantization table has exactly 256 entries, and the 8 least significant bits of the sample difference are used as index:
+Each Quantization Table contains exactly 5 Quantization Table Blocks, each block corresponding to 1 of the 5 Quantized Sample Differences, both the number of quantization steps and their distribution are stored in the bitstream. Each Quantization Table Block has exactly 256 entries, and the 8 least significant bits of the Quantized Sample Differences are used as index:
```

Comment:
> I don't have a strong opinion, but to me 'quantization table set' and 'quantization table' make sense [..]
> To me 'quantization tables sets' is less elegant, particularly since it contains data that is not only 'quantization tables set'.

+1